Core/src/main/java/com/plotsquared/core/queue/QueueCoordinator.java

```diff
@@ -412,6 +421,20 @@ public boolean enqueue() {
      */
     public abstract void setLightingMode(@Nullable LightingMode mode);
 
+    /**
+     * Get the overriding {@link SideEffectSet} to be used by the queue if it exists, else null
+     *
+     * @return Overriding {@link SideEffectSet} or null
+     */
```

Comment:
We could probably make use of the new `{@return}` inline tag, like 
```suggestion
    /**
     * {@return the overriding {@link SideEffectSet} to be used by the queue if it exists, else null}
     */
```

Reply:
Keep in mind that this feature is not supported by major IDE vendors, like IntelliJ, yet and renders as plain text.